e2e_test.go

```diff
@@ -184,13 +203,20 @@ var testsToSkip = map[string]struct{}{
 	"range_value_elision/two_params": {},
 }
 
+// groupImportTestFiles is a set of integration tests that test the
+// --disable-group-imports flag
+var groupImportTestFiles = map[string]struct{ bool }{
+	"noop_import/remove_some": {true},
+}
+
```

Comment:
If you're using this as a "set", then make the key type an empty struct type to avoid allocation.

```suggestion
// groupImportTestFiles is a set of integration tests that test the
// --disable-group-imports flag
var groupImportTestFiles = map[string]struct{}{
	"noop_import/remove_some": struct{}{},
}
```